rewrite/src/tests/test.c

```diff
@@ -65,108 +69,162 @@ static u32 PatchFunction_End(u32 index)
     *(s_functions[index].address + 1) = s_functions[index].secondNewInst;
 }
 
-static u32 PrintSVectorDiff(const char* name, const SVec3* expected, const SVec3* res)
+static u32 PrintSVectorDiff(const char* name, const SVec3* expected, const SVec3* ret)
 {
     u32 failed = 0;
     for (u32 i = 0; i < 3; i++)
     {
-        if (expected->v[i] != res->v[i])
+        if (expected->v[i] != ret->v[i])
         {
             failed = 1;
-            ND_printf("[%s] Test Failed:\nv[%d] = %d, got %d\n", name, i, expected->v[i], res->v[i]);
+            ND_printf("[%s] Test Failed:\nv[%d] = %d, got %d\n", name, i, expected->v[i], ret->v[i]);
         }
     }
     return failed;
 }
 
-static u32 PrintMatrixDiff(const char* name, const Matrix* expected, const Matrix* res, u32 cmpTrans)
+static u32 PrintMatrixDiff(const char* name, const Matrix* expected, const Matrix* ret, u32 cmpTrans)
 {
     u32 failed = 0;
     for (u32 i = 0; i < 3; i++)
     {
         for (u32 j = 0; j < 3; j++)
         {
-            if (expected->m[i][j] != res->m[i][j])
+            if (expected->m[i][j] != ret->m[i][j])
             {
                 failed = 1;
-                ND_printf("[%s] Test Failed:\nm[%d][%d] = %d, got %d\n", name, i, j, expected->m[i][j], res->m[i][j]);
+                ND_printf("[%s] Test Failed:\nm[%d][%d] = %d, got %d\n", name, i, j, expected->m[i][j], ret->m[i][j]);
             }
         }
-        if ((cmpTrans) && (expected->t.v[i] != res->t.v[i]))
+        if ((cmpTrans) && (expected->t.v[i] != ret->t.v[i]))
         {
             failed = 1;
-            ND_printf("[%s] Test Failed:\nt[%d] = %d, got %d\n", name, i, expected->t.v[i], res->t.v[i]);
+            ND_printf("[%s] Test Failed:\nt[%d] = %d, got %d\n", name, i, expected->t.v[i], ret->t.v[i]);
         }
     }
     return failed;
 }
 
 #ifdef TEST_MATH_IMPL
-void TEST_MATH_Sin(u32 angle, s32 res)
+
+void TEST_MATH_Sin(u32 angle, s32 ret)
 {
     const u32 index = PatchFunction_Beg((u32*)(&ND_MATH_Sin));
     const s32 expected = ND_MATH_Sin(angle);
-    if (expected != res) { ND_printf("[MATH_Sin] Test Failed:\nInput: %d\nExpected: %d\nResult  :%d\n", angle, expected, res); }
```

Comment:
Expected:\[space\]\[number\] vs Result:\[number\]

Reply:
it looks like result has two spaces on it now lol, probably doesn't matter though.